lib/active_admin/orm/active_record/comments/views/active_admin_comments.rb

```diff
@@ -26,7 +26,7 @@ def title
         def build_comments
           if @comments.any?
             @comments.each(&method(:build_comment))
-            div page_entries_info(@comments).html_safe, class: 'pagination_information'
+            div page_entries_info(@comments).html_safe, class: 'pagination_information', entry_name: I18n.t('active_admin.comments.resource')
```

Comment:
Haven't tried this PR but after reading the specs we have for `:entry_name`:

https://github.com/activeadmin/activeadmin/blob/6038ea6f8fd048926d59ba2c8c29ca2c40d1d41a/spec/unit/views/components/paginated_collection_spec.rb#L94-L161

it seems that this would show "Display 1 resource" instead of "Display 1 comment". Is this what you want? Is it better? How is it related to `will_paginate`? Maybe there's a bug with `will_paginate` support when infering the class name automatically (no `:entry_name` specified)...

Reply:
This is exactly what I meant about better internationalization. The only error I have ran into when trying to make it work on my project that has `will_paginate` was that one, all documentation instructions were OK, but that was never mentioned this problem and it was my case. I will update this later today with the I18n that makes it display as intended. I am not sure if it is a bug with `will_paginate`, I just read the error stack trace and fixed what it was broken in my case.

Reply:
Hi @ErvalhouS, sorry for getting back so late. What's the stacktrace you're talking about? I still don't get what problem this is fixing...